src/test/resources/testng_suites/web-saucedemo.xml

```diff
@@ -1,24 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
-<suite name="saucedemo test">
-    <test name="test login">
+<suite verbose="1" name="Saucedemo test" parallel="methods">
```

Comment:
to run tests in multiple threads add `thread-count="5"` after parallel="methods"

Reply:
this parameters also you can add back 
```
<parameter name="suiteOwner" value="qpsdemo"/>
    <parameter name="jenkinsJobName" value="ADD SOME NAME HERE/>
    <parameter name="jenkinsJobType" value="web"/>
    <parameter name="jenkinsEnvironments" value="DEMO"/>

    <parameter name="jenkinsRegressionPipeline" value="DEMO"/>

    <parameter name="jenkinsRegressionMatrix_full_regression" value="browser: chrome;"/>
    <parameter name="jenkinsAutoScreenshot" value="true"/>
```